x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -510,81 +562,75 @@ const ProcessEventDotComponents = React.memo(
               </use>
             </g>
           </svg>
-          <div
-            style={{
-              display: 'flex',
-              flexFlow: 'column',
-              left: '25%',
-              top: '30%',
-              position: 'absolute',
-              width: '50%',
-              color: NamedColors.full,
-              fontSize: `${scaledTypeSize}px`,
-              lineHeight: '140%',
-              backgroundColor: NamedColors.resolverBackground,
-              padding: '.25rem',
-            }}
+          <StyledActionsContainer
+            color={colorMap.full}
+            fontSize={scaledTypeSize}
+            topPct={actionableButtonsTopOffset}
           >
-            <div
-              id={descriptionId}
-              style={{
-                textTransform: 'uppercase',
-                letterSpacing: '-0.01px',
-                backgroundColor: NamedColors.resolverBackground,
-                lineHeight: '1',
-                fontWeight: 'bold',
-                fontSize: '0.8rem',
-                width: '100%',
-                margin: '0',
-                textAlign: 'left',
-                padding: '0',
-                color: NamedColors.empty,
-              }}
+            <StyledDescriptionText
+              backgroundColor={colorMap.resolverBackground}
+              color={colorMap.descriptionText}
+              isDisplaying={isShowingDescriptionText}
             >
               {descriptionText}
-            </div>
+            </StyledDescriptionText>
             <div
-              className={magFactorX >= 2 ? 'euiButton' : 'euiButton euiButton--small'}
-              data-test-subject="nodeLabel"
-              id={labelId}
               style={{
-                backgroundColor: labelBackground,
-                padding: '.15rem 0',
-                textAlign: 'center',
-                maxWidth: '20rem',
-                minWidth: '12rem',
-                width: '60%',
-                overflow: 'hidden',
-                whiteSpace: 'nowrap',
-                textOverflow: 'ellipsis',
-                contain: 'content',
-                margin: '.25rem 0 .35rem 0',
+                backgroundColor: colorMap.resolverBackground,
+                alignSelf: 'flex-start',
+                padding: 0,
               }}
             >
-              <span className="euiButton__content">
-                <span className="euiButton__text" data-test-subj={'euiButton__text'}>
-                  {eventModel.eventName(event)}
+              <EuiButton
+                color={labelButtonFill}
+                data-test-subject="nodeLabel"
+                fill={isLabelFilled}
+                id={labelId}
+                size="s"
+                style={{
+                  maxHeight: `${Math.min(26 + magFactorX * 3, 32)}px`,
+                  maxWidth: `${isShowingEventActions ? 400 : 210 * magFactorX}px`,
+                }}
+                tabIndex={-1}
```

Comment:
:grey_question: That works? Very cool. They didn't have tabIndex listed on https://elastic.github.io/eui/#/navigation/button but that fixes everything if it keeps it out of the tab order.